### PR TITLE
CRAN submission minor fixes/updates

### PR DESCRIPTION
**Pull request by terrytangyuan**
These are some minor fixes/updates I had to make during CRAN submission (`devtools::release` includes some additional checks).

### CODE REVIEW

File: src/interface_r/DESCRIPTION [hunk 1]
```diff
@@ -3,7 +3,9 @@ Type: Package
 Title: R Interface to 'H2O4GPU'
 Version: 0.2.0
 Authors@R: c(
-  person("Yuan", "Tang", role = c("aut", "cre"), email = "terry@h2o.ai"),
+  person("Yuan", "Tang", role = c("aut", "cre"),
+         email = "terrytangyuan@gmail.com",
+         comment = c(ORCID = "0000-0001-5243-233X")),
```
Comment (terrytangyuan) [inline src/interface_r/DESCRIPTION]: This is just an unique identifier for my involvement (many people in China have the exact same name as mine)...

File: src/interface_r/vignettes/getting_started.Rmd [hunk 2]
```diff
@@ -74,7 +74,7 @@ ce(actual = y, predicted = pred)
...
 
-The **h2o4gpu** R package does not include a suite of internal model metrics functions, therefore we encourage users to use a third-party model metrics package of their choice.  For all the examples below, we will use the [Metrics](https://cran.r-project.org/web/packages/Metrics/index.html) R package.  This package has a large number of model metrics functions, all with a very simple, unified API.
+The **h2o4gpu** R package does not include a suite of internal model metrics functions, therefore we encourage users to use a third-party model metrics package of their choice.  For all the examples below, we will use the [Metrics](https://CRAN.R-project.org/package=Metrics) R package.  This package has a large number of model metrics functions, all with a very simple, unified API.
```
Comment (terrytangyuan) [inline src/interface_r/vignettes/getting_started.Rmd]: Had to use canonical form of CRAN package url

File: src/interface_r/vignettes/getting_started.Rmd [hunk 3]
```diff
@@ -4,7 +4,7 @@ author: "Navdeep Gill, Erin LeDell, Yuan Tang"
 date: "`r Sys.Date()`"
 output: rmarkdown::html_vignette
 vignette: >
-  %\VignetteIndexEntry{Vignette Title}
+  %\VignetteIndexEntry{H2O4GPU: Machine Learning with GPUs in R}
```
Comment (terrytangyuan) [inline src/interface_r/vignettes/getting_started.Rmd]: Cannot use `Vignette Title` since it was just a placeholder from the template.

**Comment by navdeep-G** [comment]
LGTM

**Pull request merged**